roles/common/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: install opstools rpm repo configuration
```

Comment:
In fact, I don't think this pr should include anything in `roles/common` at all.